reference.md

```diff
@@ -175,18 +179,20 @@ postcondition
     Postconditions are often represented using [assertions](#assertion).
 
 precondition
-:   A condition that must be true in order for a function (or other block of code) to run correctly.
+:   A condition that must be true in order for a function (or other block of code)
+    to run correctly.
 
 regression
 :   To re-introduce a bug that was once fixed.
 
 return statement
-:   A statement that causes a function to stop executing and return a value to its caller immediately.
+:   A statement that causes a function to stop executing and return a value 
+    to its caller immediately.
 
 RGB
 :   An [additive model](#additive-color-model)
     that represents colors as combinations of red, green, and blue.
-    Each color's value is typically in the range 0..255
+    Each color's value is typically in the range 0-255
```

Comment:
What was the motivation for this change? :)

Reply:
I'm sorry, that's out of scope. I thought it was a typo as I have never seen ranges written with ".." and after a quick google it didn't seem the most common way to indicate a range. Does it have special meaning? 
Happy to change it back.

Reply:
I don't think single dash `-` indicates a range either, though. En-dash (does)[https://www.thepunctuationguide.com/en-dash.html]. To get en-dash we have to use two dashes `--` in text.
 
Regarding ranges: `git` uses two dots to indicate a range (sort of) but I'm sure that was not the reason for using two dots here.

I'd suggest reverting this change here (as the main changes have to be related to wrapping long lines) and addressing it separately.

Instead of dashes or dots, we could just use words...

Reply:
Ah yes, was going for the en-dash. But, yes you are right, words will probably make it clearer and improve readability. 
I will revert the change here.

Reply:
Thank you!